### PR TITLE
Section on null models

### LINKED ISSUE #1: Complexity v. null model
The always wise @gvdr has suggested that we could compare SVD entropy to its distribution after a null model - that's a good idea, I'll probably write the code this afternoon, in anticipation of a revision.

### PR DESCRIPTION
**Pull request by tpoisot**
- New section with the null models results
- Re-organisation of the text to merge methods and results

Closes #1

### CODE REVIEW

File: manuscript.md
Comment (TanyaS08) [inline manuscript.md]: Assuming $z_i$ should be $x_i$?

**Review by TanyaS08 — APPROVED** [review]
I like the new order/flow. Introduces the concepts (fancy maths) and then tells the story.

RE Null models: Super interesting... So even if we find that networks are very complex (based on SVD) they 'could do better'?

**Review by TanyaS08 — APPROVED** [review]

File: manuscript.md
Comment (TanyaS08) [reply manuscript.md]: Ln. 361 that is

File: manuscript.md
Comment (gvdr) [inline manuscript.md]: I wonder whether it may be appropriate to reference the more "classic" random graphs names for these models (Erdös-Renyi Gnp and Configuration model) here. This would suggest a link to a throve of established results...

**Comment by gvdr** [comment]
> RE Null models: Super interesting... So even if we find that networks are very complex (based on SVD) they 'could do better'?

Yes 👍 another complimentary way of seeing this is that although 0.995 is "very" close to 1 in absolute terms, unless we have a good understanding of what SVD entropy a "typical random network" has, 0.995 may be not surprisingly close to 1. In other words, networks (random or not) with values significantly smaller than that may be very rare.

**Comment by gvdr** [comment]
PS stocked to jump on board the manuscript. Kia ora!

File: manuscript.md
Comment (tpoisot) [reply manuscript.md]: I agree - I am quite convinced that the ecologists have re-discovered them by accident. What would you cite as the reference for each? I would dig up my copy of Newman but I'm not sure where it is...

**Comment by tpoisot** [comment]
@TanyaS08 - can you take a quick stab at the conclusion, to reflect (i) the discussion we had on zoom about assembly dynamics and (ii) the point you made about networks that can "do better"?

**Comment by TanyaS08** [comment]
So I don't know if its how it's worded in the ms that triggered this thought process or not but I was interested, since we argue that connectance constrains SVD, if this would reflect in the Z-scores when plotting them against connectance (i.e. the containing effect of connectance might explain the complexity deficit)... Not sure if my thinking checks out here (and I may be completely off the mark) but I thought maybe we would see low Z-scores for networks with a high connectance - not so sure about that though...

nullmodel_connectance

**Comment by gvdr** [comment]
That is a good idea for a check, Tanya!

The possible counterargument is that each randomised network we sample will have the same connectance (roughly or exactly) of its respective observed network. So the SVD entropy of the null model would be constrained as well. And this may explain why we don't see the expected correlation between connectance and SVD entropy z-score.

So, the way I may read your result is that the randomization (the null models) do **not** show a connectance bias.

:thinking:

**Comment by TanyaS08** [comment]
Aah I felt like there was a step missing in my logic - so if I understand correctly (in broad strokes) the random networks will still share *some* characteristics with the observed network (in this case connectance) since they are 'built off' of their respective observed network?

**Comment by gvdr** [comment]
Exactly. Null models are "change everything BUT something". Often, that something is not important, sometimes it is.

**Comment by tpoisot** [comment]
I think @gvdr is correct - the null models will have very close connectance to the empirical one, so this is an important check to have, but not one I would necessarily put into the manuscript.

@TanyaS08 , let's add the discussion on assemblage, and then we can update the preprint?

**Comment by TanyaS08** [comment]
@gvdr, would you like to have a final review before we merge to `master` and resubmit the ms? 

Also to check (for the sake of clarity and transparency), you are still onboard as being listed as a co-author as a part of the team right? 😁 I feel I also have to add your 'plain language' explanations are just the best!

**Comment by TanyaS08** [comment]
Ooh wait as a last check - we should probably tweak the abstract to reflect the new results as well?

**Comment by gvdr** [comment]
I'm very happy about the manuscript. I don't have specific comments. And I'm very very happy to be a coauthor :-)

Probably there is still opportunity for minor tweaks, but nothing major. I may go for a re-read over the weekend, if necessary, but if you want to proceed more speedily go for it!

**Comment by TanyaS08** [comment]
Okay cool - I'll merge this to master then and maybe we aim to resubmit early next week? 🐜 @tpoisot I guess it makes sense to hold off updating the preprint until then as well - but if the desire is there I guess we could go for that

**Pull request merged**
